builtin/providers/google/provider_test.go

```diff
@@ -38,18 +39,40 @@ func testAccPreCheck(t *testing.T) {
 		os.Setenv("GOOGLE_CREDENTIALS", string(creds))
 	}
 
-	if v := os.Getenv("GOOGLE_CREDENTIALS"); v == "" {
-		if w := os.Getenv("GOOGLE_CLOUD_KEYFILE_JSON"); w == "" {
-			t.Fatal("GOOGLE_CREDENTIALS or GOOGLE_CLOUD_KEYFILE_JSON must be set for acceptance tests")
+	multiEnvSearch := func(ks []string) string {
+		for _, k := range ks {
+			if v := os.Getenv(k); v != "" {
+				return v
+			}
 		}
+		return ""
 	}
 
-	if v := os.Getenv("GOOGLE_PROJECT"); v == "" {
-		t.Fatal("GOOGLE_PROJECT must be set for acceptance tests")
+	creds := []string{
+		"GOOGLE_CREDENTIALS",
+		"GOOGLE_CLOUD_KEYFILE_JSON",
+		"GCLOUD_KEYFILE_JSON",
+	}
+	if v := multiEnvSearch(creds); v == "" {
+		t.Fatalf("One of %s must be set for acceptance tests", strings.Join(creds, ", "))
 	}
 
-	if v := os.Getenv("GOOGLE_REGION"); v != "us-central1" {
-		t.Fatal("GOOGLE_REGION must be set to us-central1 for acceptance tests")
+	projs := []string{
```

Comment:
Hey @stack72 

- creds - places to look for credentials json
- projs - places to look for project
- regs - places to look for regions

We could make this a constant and share it between the provider and provider_test, but that wasn't a pattern I saw elsewhere in the codebase and didn't want to introduce a new paradigm. This is just the test file searching for the place to pull in creds from, and it arguably could remain unchanged.